src/util/support.js

```diff
@@ -0,0 +1,3 @@
+const div = document.createElement('div');
+export const shadowDomV0 = !!div.createShadowRoot;
```

Comment:
That's shouldn't be necessary because if there's a v0 polyfill, then this should work. If not, that's a problem with their polyfill.